app/public/javascripts/timer.js

```diff
@@ -0,0 +1,95 @@
+class Timer {
+    constructor(root) {
+      root.innerHTML = Timer.getHTML();
+  
+      this.el = {
+        minutes: root.querySelector(".timer__part--minutes"),
+        seconds: root.querySelector(".timer__part--seconds"),
+        control: root.querySelector(".timer__btn--control"),
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 8.](https://app.codacy.com/gh/cse110-fa21-group17/cse110-fa21-group17/pullRequest?prid=8445086)
```suggestion
      control: root.querySelector(".timer__btn--control"),
```